lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -276,8 +277,12 @@ function reportAssertion(assertion) {
   RegExp.prototype.toJSON = RegExp.prototype.toString;
 
   if (assertion.equal) {
-    console.log(`  ${log.greenify(log.tick)} ${assertion.category}: ` +
-        log.greenify(assertion.actual));
+    if (assertion.actual !== null && typeof assertion.actual === 'object') {
```

Comment:
super nit but mind flipping the order of these to `is an object && is not null`? took me a bit to realize this was just an object check

Reply:
can we just use a helper method like so?  

```js
const isPlainObject = function (obj) {
	return Object.prototype.toString.call(obj) === '[object Object]';
};
```

Reply:
> `Object.prototype.toString.call(obj) === '[object Object]';`

what year is it?

Reply:
:P